docs/compliance.md

```diff
@@ -0,0 +1,17 @@
+# Compliance
+
+We use the [Checkov](https://www.checkov.io/) static analysis tool to check for compliance with infrastructure policies.
+
+## Setup
+
+To run this tool locally, first install Checkov by running the following command.
+
+```bash
+brew install checkov
+```
+
+## Check compliance
+
+```bash
+make infra-check-compliance
+```
```

Comment:
Could the checkov check be added to a .pre-commit-config.yaml?

Reply:
yeah totally. that said, i might be in the minority on this but i'm actually not a fan of checking in precommit hooks into the repo as they impose restrictions on a developer's ability to develop the way they want to on their own branch (personally i think running checks on precommit makes commits too slow and makes it harder for me to have lots of small atomic commits). i think pull requests are the right time to impose team level checks and restrictions. a developer could add the precommit hook themselves (and the provided make commands should make it easy to do so). what do you think about the idea of adding a `docs/pre-commit.md` file that explains how to add pre-commit hooks into your own workflow as an option developers can take but aren't forced to?

Reply:
Agreed that precommit hooks can stifle a quick workflow. Could be in that docs/pre-commit.md or an example snippet for adding checkov to the .pre-commit-config.yaml in this compliance doc.

Reply:
good idea about adding to this compliance doc i'll do that for now to keep the scope small